resources/sass/app.scss

```diff
@@ -250,6 +250,10 @@ dl.import-meta dd:not(:last-child) {
     padding-bottom: 16px;
 }
 
+dl.import-meta .download-csv {
+    float:right;
```

Comment:
nitpick (non-blocking): missing space after the colon

(Though we really ought to have linting for that)

Reply:
fixed!